hedera-node/src/test/java/com/hedera/services/txns/validation/ContextOptionValidatorTest.java

```diff
@@ -604,8 +607,38 @@ void memoCheckWorks() {
 	}
 
 	@Test
-	void rejectsInvalidBatchSize(){
+	void rejectsInvalidBurnBatchSize(){
 		var list = LongStream.range(0, 1000).boxed().collect(Collectors.toList());
 		assertEquals(BATCH_SIZE_LIMIT_EXCEEDED, subject.maxBatchSizeBurnCheck(list.size()));
 	}
+
+	@Test
+	void rejectsInvalidNftTransfersSize() {
+		var ls = LongStream.range(0, 10000).boxed().collect(Collectors.toList());
+		assertEquals(BATCH_SIZE_LIMIT_EXCEEDED, subject.maxNftTransfersLenCheck(ls.size()));
+	}
+
+	@Test
+	void rejectsInvalidWipeBatchSize() {
+		var ls = LongStream.range(0, 10000).boxed().collect(Collectors.toList());
+		assertEquals(BATCH_SIZE_LIMIT_EXCEEDED, subject.maxBatchSizeWipeCheck(ls.size()));
+	}
+
+	@Test
+	void rejectsInvalidMintBatchSize() {
+		var ls = LongStream.range(0, 10000).boxed().collect(Collectors.toList());
+		assertEquals(BATCH_SIZE_LIMIT_EXCEEDED, subject.maxBatchSizeMintCheck(ls.size()));
+	}
+
+	@Test
+	void rejectsInvalidQueryRange() {
+		var ls = LongStream.range(0, 10000).boxed().collect(Collectors.toList());
+		assertEquals(QUERY_RANGE_LIMIT_EXCEEDED, subject.nftMaxQueryRangeCheck(-10, 100000));
+	}
+
+	@Test
+	void rejectsInvalidMetadata() {
+		var metadata = ByteString.copyFromUtf8(LongStream.range(0, 10000).boxed().collect(Collectors.toList()).toString());
+		assertEquals(METADATA_TOO_LONG, subject.nftMetadataCheck(metadata.toByteArray()));
+	}
```

Comment:
I don't think that we need the `ls` variable for the tests.
I think that we need to mock the `dynamicProperties.*` methods for each of them as-well